ChatRPG/ChatRPG/API/SallingClient.cs

```diff
@@ -0,0 +1,55 @@
+using RestSharp;
+using RestSharp.Authenticators;
+
+namespace ChatRPG.API;
+
+public class SallingClient : ISallingClient, IDisposable
+{
+    private const string _sallingBaseUrl = "https://api.sallinggroup.com/v1/";
+    private readonly ILogger<SallingClient> _logger;
+    private readonly RestClient _client;
+
+    public SallingClient(ILogger<SallingClient> logger, IConfiguration configuration)
+    {
+        _logger = logger;
+
+        var options = new RestClientOptions(_sallingBaseUrl)
+        {
+            Authenticator = new JwtAuthenticator(configuration.GetSection("ApiKeys").GetValue<String>("Salling")),
+            FailOnDeserializationError = false
+        };
+        _client = new RestClient(options);
```

Comment:
den skal helst kunne tage imod en `HttpMessageClient` som [vist her](https://restsharp.dev/v107/#mocking) for at muliggøre testing, det må vi lige undersøge

måske kan den bare tage imod en `IRestClient` i constructoren, og så sætter man den op i Main.cs

Reply:
Good call, den tager jeg med

Reply:
added to llm client, resolving